nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/NuxeoCloudClient.java

```diff
@@ -66,17 +66,17 @@
     public static final String DATASET_TEMPLATE = "{\n" +
             "  \"entity-type\": \"document\",\n" +
             "  \"name\": \"%s\",\n" +
-            "  \"type\": \"AI_Corpus\",\n" +
+            "  \"type\": \"DatasetExport\",\n" +
             "  \"properties\": {\n" +
             "    \"dc:title\": \"%s\",\n" +
-            "    \"ai_corpus:documents_count\": %s,\n" +
```

Comment:
This is the document on the cloud.  It needs to be "ai_corpus", don't change this.

Reply:
ok